app/Jobs/StoreJob.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace App\Jobs;
+
+class StoreJob extends AbstractOwnedJob
+{
+    /**
+     * Create a new job instance.
+     *
+     * @return void
+     */
+    public function __construct($hoqu_job, $job_name, $geometry)
```

Comment:
Non avrei messo geometry ma input: il fatto che sia geometry è un caso specifico del caso che stiamo gestendo, ma in generale è una stringa json che il processor saprà gestire

Reply:
Qui ho messo così perchè altrimenti è necessaria la validazione dell'input dentro il job ...

Reply:
Corretto la eventuale validazione la fa il JOB: lo vedremo nella astrazione prevista (non ne abbiamo parlato in dettaglio) 
getInput
ValidateInput
processInput
buildOutput
sendOutputToHoqu (api Done)